lib/ui/page/home/page/my_profile/controller.dart

```diff
@@ -182,6 +179,12 @@ class MyProfileController extends GetxController {
   /// Returns the current [Credentials].
   Rx<Credentials?> get credentials => _authService.credentials;
 
+  /// Private flag for returning early from [onObserve] callback
+  bool _isIgnoreObserving = false;
```

Comment:
According to the [Contribution Guide](https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#classes-constructors-fields-and-methods-ordering), the fields should be sorted before the getters/setters.

Reply:
That's right, i'll correct it

Reply:
moved it before getters as in Contribution Guide, can't move it to Init method cause onObserve now is function getter.

fixed on 3bdb4c364a9c19145ac3970c4389cdc034f8f30f commit